File: labs/01/pl/calcula_final.c
//en C

#include <stdio.h>
#include <stdlib.h>

int nota_del_final(int a, int b){
    return (110 - 3*a - 3*b) / 4;
}

int main(int argc, char *argv[]){ //el primero cantidad(count) y el segundo los argumentos
    
  if(argc != 3){ //argc es 3 = 2 argumentos + name
      printf("Debe ingresar exactamente 2 notas\r\n");
      return 0;
  }

  int x = atoi(argv[1]); //guardamos en la primera
  int y = atoi(argv[2]); //atoi convierte cadena a entero
  //ya que lo recibe como ascii o utf-8 convertimos al numero

  printf("Me voy por: %d", nota_del_final(x, y));
}

File: labs/01/pl/calcula_final.py
#en python

import sys #para recibir parametros de entrada

def nota_del_final(a, b):
  res = (110 - 3*a - 3*b) / 4
  print("Calculando por cuánto me voy:", res)
  return res

if len(sys.argv) != 3: #siempre la mimsa funcion es un parametro asi serian 2 + name
  print("Debe ingresar exactamente 2 notas") #de no recibir 2 mostrar esto
  sys.exit(0) #se acaba

x = int(sys.argv[1]) #guardamos el primero  cast de entero para pasarlo a entero
y = int(sys.argv[2]) #guardamos el segundo

resultado = nota_del_final(x, y)
print(resultado)
